docs/getting-started-guides/rkt/index.md

```diff
@@ -70,110 +65,147 @@ $ cat <<EOF >/etc/rkt/net.d/k8s_flannel_example.conf
-  Leave empty to use the default one (`localhost:15441`).
-- `--rkt-path=$PATH_TO_RKT_BINARY` sets the path of rkt binary.
-  Leave empty to use the first rkt in $PATH.
-- `--rkt-stage1-image` sets the name of the stage1 image, e.g. coreos.com/rkt/stage1-coreos.
-  Leave empty to use the default stage1 image in the rkt's configuration.
+* `--container-runtime=rkt` Set the node's container runtime to rkt.
+* `--rkt-api-endpoint=HOST:PORT` Set the endpoint of the rkt API service. Default: `localhost:15441`.
+* `--rkt-path=PATH_TO_RKT_BINARY` Set the path of the rkt binary. Optional. If empty, look for `rkt` in `$PATH`.
+* `--rkt-stage1-image=STAGE1` Set the name of the stage1 image, e.g. `coreos.com/rkt/stage1-coreos`. Optional. If not set, the default Linux kernel software isolation stage1 is used.
 
-If you are using the [hack/local-up-cluster.sh](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/hack/local-up-cluster.sh) script to launch the local cluster, then you can edit the environment variable `CONTAINER_RUNTIME`, `RKT_PATH` and `RKT_STAGE1_IMAGE` to set these flags, the `RKT_PATH` and `RKT_STAGE1_IMAGE` are optional if you have `rkt` in your `$PATH` with appropriate configuration.
+If you are using the [hack/local-up-cluster.sh](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/hack/local-up-cluster.sh) script to launch the cluster, you can edit the environment variables `CONTAINER_RUNTIME`, `RKT_PATH`, and `RKT_STAGE1_IMAGE` to set these flags. `RKT_PATH` and `RKT_STAGE1_IMAGE` are optional if `rkt` is in `$PATH` with appropriate configuration.
 
 ```shell
 $ export CONTAINER_RUNTIME=rkt
-$ export RKT_PATH=$PATH_TO_RKT_BINARY
-$ export RKT_STAGE1_IMAGE=$NAME_OF_THE_STAGE1_IMAGE
+$ export RKT_PATH=<rkt_binary_path>
+$ export RKT_STAGE1_IMAGE=<stage1-name>
 ```
 
-Then we can launch the local cluster using the script:
+Now you can launch the cluster using the `local-up-cluster.sh` script:
 
 ```shell
 $ hack/local-up-cluster.sh
 ```
 
-We are also working on setting up rkt as the container runtime for [minikube](https://github.com/kubernetes/minikube/issues/168).
+We are also working on getting rkt working as the container runtime in [minikube](https://github.com/kubernetes/minikube/issues/168).
+
+### Launch a rktnetes cluster on Google Compute Engine (GCE)
 
-### Launch a CoreOS/rkt cluster on Google Compute Engine (GCE)
+This section outlines using the `kube-up` script to launch a CoreOS/rkt cluster on GCE.
 
-Here we provide instruction on how to use the `kube-up` script to launch a CoreOS/rkt cluster on GCE.
-In order to do that, you need to specify the OS distribution, project, image:
+First, specify the OS distribution, your GCE project id, and the instance images for the master and nodes. Set the `KUBE_CONTAINER_RUNTIME` to `rkt`:
 
 ```shell
 $ export KUBE_OS_DISTRIBUTION=coreos
-$ export KUBE_GCE_MASTER_PROJECT=coreos-cloud
+$ export KUBE_GCE_MASTER_PROJECT=<my-proj-id>
```

Comment:
These projects should be left as `coreos-cloud` if it's using `OS_DISTRIBUTION=coreos`. This is the project the OS image is taken from; the project instances are launched in is not set here, but rather in your `gcloud` config.